iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -6119,21 +6123,22 @@
 					"$(OMIM_ROOT)/3party/jansson/src",
 				);
 				INFOPLIST_FILE = OMaps.plist;
-				LIBRARY_SEARCH_PATHS = (
-					"$(inherited)",
-				);
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
+				LIBRARY_SEARCH_PATHS = "$(inherited)";
 				OTHER_CFLAGS = (
 					"$(inherited)",
 					"-Wall",
 					"-Wno-unused-local-typedef",
+					"-Wno-nonnull",
+					"-Wno-nullability-completeness",
```

Comment:
Are you sure about this?

Reply:
Please also revert these 2 lines. Zero-warnings policy is done by fixing the source of warnings, not by silencing them.